tests/3-unit/PluginFormcreatorForm.php

```diff
@@ -1313,4 +1313,26 @@ public function testAdd() {
       $row = array_shift($rows);
       $this->string($row['name'])->isEqualTo(\PluginFormcreatorSection::getTypeName(1));
    }
+
```

Comment:
Are these tests cases commented by mistake ?
Why not remove them completely ?

Reply:
These unit  tests are commented out because they are not ready yet. This is a 1st attempt to design the cases provider.